README.md

```diff
@@ -139,11 +139,9 @@ Then edit the newly created scheme to make it use a different env file. From the
 - Click "Pre-actions", and under the plus sign select "New Run Script Action"
 - Where it says "Type a script or drag a script file", type:
   ```
-  echo ".env.staging" > /tmp/envfile   # replace .env.staging for your file
+  cp ${PROJECT_DIR}/../.env.staging .env  # replace .env.staging for your file
```

Comment:
1. Shouldn't the destination be ${PROJECT_DIR}/../.env then?
2. there is another reference to "/tmp/envfile" up the text
3. in terms of implementation /tmp/envfile takes priority, however this is nowhere documented; I would expect either to document it or to remove it from the code entirely

Reply:
The `/tmp/envfile` priority is definitely confusing especially if switching to the new recommendation on a CI server where past builds populated `/tmp/envfile`.